modules/Cargo.toml

```diff
@@ -17,6 +17,7 @@ description = """
 # This feature grants access to development-time mocking libraries, such as `MockContext` or `MockHeader`.
 # Depends on the `testgen` suite for generating Tendermint light blocks.
 mocks = [ "tendermint-testgen", "sha2" ]
+test_no_std = []
```

Comment:
Let's feature gate the `std::error::Error` mode as the `"std_err"` feature. We can later set the `"std"` feature to enable the `"std_err"` feature.